content/Reducing_Power_Consumption.md

```diff
@@ -8,7 +8,7 @@ The HALT instruction should be used whenever possible to reduce power
 consumption & extend the life of the batteries. This command stops the
 system clock, reducing the power consumption of both the CPU and ROM.
 
-The CPU will remain stopped until an interrupt occurs at which point the
+The CPU will remain stopped until an interrupt *enabled in the IE Register (FFFF)* occurs at which point the
```

Comment:
I'd rather word it as "enabled *by* the IE register", and if possible add an anchor link to the section in question.

Reply:
How should I add the anchor link ?

Reply:
`*enabled in [the IE register ($FFFF)](#ffff-ie-interrupt-enable-r-w)*` should do the trick.

Reply:
👍